package.json

```diff
@@ -33,10 +33,10 @@
     "zone.js": "^0.6.23"
   },
   "devDependencies": {
+    "@angular/cli": "^1.0.0",
     "@angular/compiler-cli": "2.4.0",
```

Comment:
And one more thing @13nehu ... Squash all your commits into one. (How to squash -> https://davidwalsh.name/squash-commits-git)